src/cf-layout.less

```diff
@@ -1403,6 +1403,201 @@
+                        <div class="featured-content-module_wrapper">
+                        <img class="featured-content-module_img" src="http://placekitten.com/g/540/270">
+                        </div>
+                      </div>
+                    </section>
+                </section>
+            </div>
+        </main>
+      notes:
+        - "Featured content module, like a hero, consists of headline/text/optional call to action
+           along with a visual. It is intended to be used in a main content column next to a   
+           sidebar."
+        - "Text is full width & displayed above the visual in the default/mobile view. At larger
+           screen sizes, the text occupies a fixed portion of the screen (equal to the width of 
+           5 of 12 columns at 799px for tablets & 3 of 12 columns at 1140px for desktop). The
+           visual occupies the remaining space. The visual also has a fixed width on larger screens   
+           (defaults to 540 so it has 16x9 ratio on desktop where the height of the module is 
+           270px), and is center-cropped when it becomes too wide for the available space."
+        - "Add the .featured-content-module__left modifier when the sidebar column is on the right 
+           side. The background color will bleed off the left edge of the screen at desktop sizes."
+      codenotes:
+        - .featured-content-module
+        - .featured-content-module__left
+  tags:
+    - cf-layout
+*/
+
+
+.featured-content-module {
+  background-color: @gray-5;
+  border-top: 1px solid @gray-50;
+  border-bottom: 1px solid @gray-50;
+  position: relative;
+  
+  .respond-to-max(@mobile-max, {    
+    margin-right: unit((-@grid_gutter-width / 2) / @base-font-size-px, em);
+    margin-left: unit((-@grid_gutter-width / 2) / @base-font-size-px, em);
+  });
+  
+  .respond-to-min(@tablet-min, {    
+    margin-right: unit(-@grid_gutter-width / @base-font-size-px, em);
+    margin-left: unit(-@grid_gutter-width / @base-font-size-px, em);
+  });
+  
+  .respond-to-min(800px, {
+    box-sizing: border-box;
+    min-height: 270px;
+    margin-right: 0;
+    border-right: 1px solid @gray-50;
+    padding-left: unit(@grid_gutter-width / @base-font-size-px, em);
+  });
+  
+  &__left {
+    .respond-to-min(800px, {
+      margin-right: unit(-@grid_gutter-width / @base-font-size-px, em);
+      margin-left: 0;
+      padding-left: 0;
+      border-right-width: 0;
+      
+      // bleed background off left edge of page
+      &:before {
+        left: -200%;
+        background-color: @gray-5;
+        content: '';
+        display: block;
+        position: absolute;
+        width: 200%;
+        border-top: 1px solid @gray-50;
+        border-bottom: 1px solid @gray-50;
+        top: -1px;
+        bottom: -1px;
+      }
+    });
+  }
+  
+  
+  &_text {
+    padding-top: unit(25px / @base-font-size-px, em); // approximates 30px visually
```

Comment:
Consider instead:
`unit((@grid_gutter-width - 5px) / @base-font-size-px, em); // Eyeballed optical offset to 30px visually.`

"Eyeball" and "optical" are both code comments I've seen in the majority of these visually offset cases. In order to find them should they need to change, it's best to try to format them the same. For the same reason, using the `@grid_gutter-width` variable minus and offset shows this is intended to be the `@grid_gutter-width` and not an arbitrary `25px` value.